packages/monaco/src/browser/monaco-text-model-service.ts

```diff
@@ -127,12 +127,15 @@ export class MonacoTextModelService implements ITextModelService {
 
     protected readonly modelOptions: { [name: string]: (keyof ITextModelUpdateOptions | undefined) } = {
         'editor.tabSize': 'tabSize',
-        'editor.insertSpaces': 'insertSpaces'
+        'editor.insertSpaces': 'insertSpaces',
+        'editor.indentSize': 'indentSize'
     };
 
     protected toModelOption(editorPreference: EditorPreferenceChange['preferenceName']): keyof ITextModelUpdateOptions | undefined {
         switch (editorPreference) {
             case 'editor.tabSize': return 'tabSize';
+            // @monaco-uplift: uncomment this line once 'editor.indentSize' preference is available
+            //  case 'editor.indentSize': return 'tabSize';
```

Comment:
Minor: Given that the previous question resulted in `indentSize`, it should be here the same as well.

```suggestion
            //  case 'editor.indentSize': return 'indentSize';
```

Reply:
Fixed with new commit.